composer.json

```diff
@@ -21,20 +15,20 @@
         }
     ],
     "require": {
-        "php": "^5.6 || ^7.0",
-        "guzzlehttp/guzzle": "^6.1",
+        "php": "^7.0",
+        "php-http/guzzle6-adapter": "^2.0",
```

Comment:
Optional? To my understanding of composer this is a requirement hence there is no benefit for people using this repo as a dependency in their projects.
IMO we should remove the adapter completely.
For generation of the archives we should self-require this repo and the adapter and build the archives from this "intermediate build project".

Reply:
> Optional? To my understanding of composer this is a requirement hence there is no benefit for people using this repo as a dependency in their projects.
> IMO we should remove the adapter completely.
> For generation of the archives we should self-require this repo and the adapter and build the archives from this "intermediate build project".

According to the `composer.json` file of this repository, this is no standalone project and there are some projects which are using the as a library: https://packagist.org/packages/heise/shariff/dependents

>  IMO we should remove the adapter completely.
I removed the adapter from the composer file, but it will be added to the archive when calling `build.sh`. So you have an archive which you can use as a standalone project.